service/src/main/java/co/cask/wrangler/DataPrep.java

```diff
@@ -44,14 +43,9 @@ public void configure() {
     setDescription("DataPrep Backend Service");
 
     createDataset(DirectivesService.WORKSPACE_DATASET, WorkspaceDataset.class,
-                  DatasetProperties.builder().setDescription("DataPrep Dataset").build());
-    try {
-      createDataset(RecipeService.DATASET, ObjectMappedTable.class,
-                    ObjectMappedTableProperties.builder()
-                    .setType(RecipeDatum.class).build());
-    } catch (UnsupportedTypeException e) {
-      throw new RuntimeException("Unable to create Recipe store dataset. " + e.getMessage());
-    }
+                  DatasetProperties.builder().setDescription("DataPrep Directive Store").build());
+    createDataset(DATAPREP_DATASET, Table.class,
+                  DatasetProperties.builder().setDescription("DataPrep All Store").build());
```

Comment:
Moving everything to one dataset. Don't want to have multiple datasets. Plan is to migrate everything to this dataset.